.gitignore

```diff
@@ -255,3 +255,6 @@ bazel_auth.rc
 
 # ctest
 /Testing/
+
+# Meson
+.meson-subproject*
```

Comment:
Is this a standard folder Meson creates?

Reply:
Yeah, meson generates those and is consistent for every subproject I've seen and used whether it be meson straight up or a meson -> cmake combo.

Reply:
I guess that could be `.meson-subproject*.txt` since only txt files are created.  I just go the catchall route to be safe.